docs/cncf_conformance.md

```diff
@@ -0,0 +1,167 @@
+
+* A Linux or MacOS machine with
+  [Terraform 0.11.x](https://www.terraform.io/downloads.html) installed.
+* A [Google Cloud](docs/gcp.md), [AWS](docs/aws.md) or [Azure](docs/azure.md)
+  account with enough permissions to provide the needed infrastructure
+
+## Preparing the infrastructure
+
+**NOTE:** These instructions are targeted at a Google Cloud Platform deployment.
+
+Start by cloning this repo:
+
+```
+$ git clone git@github.com:mesosphere/dcos-kubernetes-quickstart.git
+$ cd dcos-kubernetes-quickstart
+```
+
+Then generate the default infrastructure configuration:
+
+```
+$ make gcp
+```
+
+This will output sane defaults to `.deploy/desired_cluster_profile`. Now, edit
+said file and set the `gcp_project` and the `gce_ssh_pub_key_file` variables.
+Please, do not set a smaller instance (VM) type on the risk of failing to
+install Kubernetes. In the end, the `.deploy/desired_cluster_profile` file
+should look like this:
+
+```
+custom_dcos_download_path = "https://downloads.dcos.io/dcos/stable/1.11.0/dcos_generate_config.sh"
+num_of_masters = "1"
+num_of_private_agents = "3"
+num_of_public_agents = "1"
+#
+gcp_project = "<project-id>"
+gcp_region = "us-west1"
+gcp_ssh_pub_key_file = "<path-to-ssh-public-key>"
+#
+# If you want to use GCP service account key instead of GCP SDK
+# uncomment the line below and update it with the path to the key file
+#gcp_credentials_key_file = "/PATH/YOUR_GCP_SERVICE_ACCOUNT_KEY.json"
+#
+gcp_bootstrap_instance_type = "n1-standard-1"
+gcp_master_instance_type = "n1-standard-8"
+gcp_agent_instance_type = "n1-standard-8"
+gcp_public_agent_instance_type = "n1-standard-8"
+#
+# Change public/private subnetworks e.g. "10.65." if you want to run multiple clusters in the same project
+gcp_compute_subnetwork_public = "10.64.0.0/22"
+gcp_compute_subnetwork_private = "10.64.4.0/22"
+# Inbound Master Access
+admin_cidr = "0.0.0.0/0"
+
+# Uncomment the line below if you want short living cheap cluster for testing
+#gcp_scheduling_preemptible = "true"
+```
+
+Now, launch the DC/OS cluster by running
+
+```
+$ make get-cli launch-dcos setup-cli
+```
+
+This command will:
+
+1. Download the `dcos` CLI and `kubectl` to your machine.
+1. Provision the necessary infrastructure in GCP and install DC/OS.
+1. Setup the `dcos` CLI to access the newly created DC/OS cluster.
+
+As part of the last step, your browser will open and ask you to login with
+a Google, GitHub or Microsoft account. Choose an option and copy the resulting
+OpenID token to the shell where you ran the abovementioned command.
+
+## Installing `dcos-kubernetes`
+
+To install `dcos-kubernetes` in the newly created DC/OS cluster run
+
+```
+$ dcos package install --yes --options=./resources/options-ha.json kubernetes
+```
+
+Wait until all tasks are running before proceeding. You can track installation
+progress using
+
+```
+$ watch dcos kubernetes plan show deploy
+```
+
+When installation is successful you will see the following output:
```

Comment:
When the installation has successfully completed, you should see the following output: